design/predict_interval/predict_interval.Rmd

```diff
@@ -0,0 +1,76 @@
+---
+title: "predict intervals"
```

Comment:
this still applies I think - since we don't have a separate design PR now, just merge this content directly into vignettes/predict.Rmd and remove this one

Reply:
it is combined into the vignettes with a section "Parametric Sampling for prediction interval", mainly covers the conditional variance formula.